### DISCUSSION TITLE
there seems to be a mismatch between whats on pypi and conda-forge

### DISCUSSION THREAD

**Discussion by jgunstone** [manzt/anywidget#704] [General]
![image](https://github.com/user-attachments/assets/1da674c0-2a45-4199-bdf9-4c9e318462b5)

![image](https://github.com/user-attachments/assets/a68d0f39-582a-449e-96e6-2f0122919f30)

maybe because this one is out of step with the others....? 

![image](https://github.com/user-attachments/assets/d94b11a6-de9b-443a-ab23-276d03955dac)

**Comment by manzt** [comment]
anywidget is a monorepo with several different [packages]( https://github.com/manzt/anywidget?tab=readme-ov-file#packages). conda-forge mirrors releases to pypi (https://pypi.org/project/anywidget/) which come from the anywidget tag. create-anywidget is a cli we we publish to npm for bootstrapping new anywidget projects. It has separate version and releases.

**Comment by jgunstone** [comment]
gotcha - my mistake
